list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
**Observations**: The file directly outputs data without using any data sanitization.

**Recommendations**: Use `htmlspecialchars()` to prevent XSS when outputting data from the database.

**Example**:
```php
<td><?php echo htmlspecialchars($row['title']); ?></td>
<td><?php echo htmlspecialchars($row['author']); ?></td>
<td><?php echo htmlspecialchars($row['year']); ?></td>
```